src/msspreadsheets/spreadsheet_constants.bal

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//HTTP Status Codes
+const int HTTP_STATUS_OK = 200;
+const int HTTP_STATUS_CREATED = 201;
+const int HTTP_STATUS_NO_CONTENT = 204;
+
+//API urls
+const string BASE_URL = "https://graph.microsoft.com/v1.0/me/drive/root";
+
+// Error Codes
```

Comment:
```suggestion
// Error Codes.
```

Reply:
Based on the code level changes requested in this PR we had to remove the above mentioned line.